public/components/organisms/footer.html

```diff
@@ -1,85 +1,163 @@
+      </div>
+
+      <div>
+        <ul class="main_column">
+          <li>
+            <a href="#">Cohorts and Graduates</a>
+          </li>
+          <li>
+            <a href="#">Photo Albums</a>
+          </li>
+        </ul>
       </div>
-      <div class="footer_social footer-column">
-        <h4>Connect with us!</h4>
-        <ul class="footer_social-icons">
-          <li><a href="http://www.facebook.com"><img src='../../img/icon_facebook.png' /></a></li>
-          <li><a href="http://www.twitter.com"><img src='../../img/icon_instagram.png' /></a></li>
-          <li><a href="http://www.youtube.com"><img src='../../img/icon_twitter.png' /></a></li>
+    </div>
+
+    <div>
+      <hr class="hr-2" />
+    </div>
+
+    <div class="footer-main-2">
+      <h2>Engage</h2>
+
+      <div>
+        <ul class="main_column">
+          <h3>Participants</h3>
+          <li>
+            <a href="#">Student Application</a>
+          </li>
+        </ul>
+      </div>
+
+      <div>
+        <ul class="main_column">
+          <h3>Volunteers</h3>
+          <li>
+            <a href="#">Volunteer Opportunities </a>
+          </li>
+          <li>
+            <a href="#">Volunteer Application</a>
+          </li>
+        </ul>
+      </div>
+
+      <div>
+        <ul class="main_column">
+          <h3>Supporters</h3>
+          <li>
+            <a href="#">Join the Refcode Extended Professional Network</a>
+          </li>
+          <li>
+            <a href="#">Subscribe to our Newsletter</a>
+          </li>
+          <li>
+            <a href="#">Donate</a>
+          </li>
         </ul>
       </div>
-    </footer>
-  </markup>
-  <includes>
-     <link href="https://fonts.googleapis.com/css2?family=Lexend+Deca&display=swap" rel="stylesheet">
-  </includes>
+
+      <div>
+        <ul class="main_column">
+          <h3>Businesses</h3>
+          <li>
+            <a href="#">Partner with us</a>
+          </li>
+        </ul>
+      </div>
+    </div>
+
+    <div class="footer-main-3">
+      Refcode in partnership with
+      <a class="underline-footer" href="https://tektontraining.org/">
+        Tektan Career Training
+      </a>
+      is a 501(c)3 nonprofit
+    </div>
+  </footer>
 </div>
+<style>
+  /* =================================
+    Base Layout Styles
+  ==================================== */
+  .footer {
+    background: white;
+    color: black;
+  }
+
+  .footer-container {
+    display: grid;
+    column-gap: 40px;
+  }
+
+  .footer-main-1,
+  .footer-main-2 {
+    display: grid;
+    grid-template-columns: repeat(5, 1fr);
+    column-gap: 40px;
+    margin-left: 75px;
+  }
+
+  .footer-main-3 {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    font-size: 12px;
+  }
+
+  .underline-footer {
+    text-decoration: underline;
+    margin-left: 5px;
+    margin-right: 5px;
+  }
+
+  li {
+    list-style-type: none;
+    margin: 0 0 12px 0;
+  }
```

Comment:
The way this is written, it will apply to _every `li` on the entire site_. We want to prevent this style from "leaking out" to other parts of the site. To keep it contained to the footer, you can re-write it like this:

```lang=css
.footer li {
    list-style-type: none;
    margin: 0 0 12px 0;
  }
  ```
All of the CSS selectors that are not using a unique class need to be prefixed with that `.footer `, including the `a`, and `h2` below.